### PR TITLE
Snapshotting dev to 6.x

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

**Comment by rakatyal** [comment]
@omefire , @riknoll : Please review.

File: www/docs/en/6.x/config_ref/images.md [hunk 1]
```diff
@@ -19,17 +19,17 @@ license: >
 
 title: Customize app icons
```
Comment (omefire) [inline www/docs/en/6.x/config_ref/images.md]: 'app' -> 'application'

File: www/docs/en/6.x/config_ref/images.md [hunk 2]
```diff
@@ -137,15 +140,17 @@ For Windows the recommended approach to define app icons is to use `target` attr
     </platform>
 ```
 
-where `source` is the path to the icon which needs to be added.
+where `src` is the path to the icon which needs to be added.
+
+The Windows platform handles MRT icons automatically, so if you specify `src="res/windows/storelogo.png"` the following files will be copied into the application's `images` folder: `res/windows/storelogo.scale-100.png`, `res/windows/storelogo.scale-200.png`, etc.
 
-Please note that Windows platform handles MRT icons automatically, so if you specify `src="res/windows/storelogo.png"` the following files will be copied into app's `images` folder: `res/windows/storelogo.scale-100.png`, `res/windows/storelogo.scale-200.png`, etc.
+TODO Define what MRT is.
```
Comment (omefire) [inline www/docs/en/6.x/config_ref/images.md]: Should this go in a JIRA ?

File: www/docs/en/6.x/config_ref/images.md [hunk 3]
```diff
@@ -137,15 +140,17 @@ For Windows the recommended approach to define app icons is to use `target` attr
...
 
-The `target` attribute specifies the base name for resultant icons. For every icon file destination filename is calculated as `target + '.' + MRT_qualifiers + extension(src)`. For the icons to display properly in resultant app every `target` value should be the one of icon filenames, defined in application's `.appxmanifest` file.
+The `target` attribute specifies the base name for the resultant icons. For every icon file, its destination filename is calculated as `target + '.' + MRT_qualifiers + extension(src)`. For the icons to display properly in the application, every `target` value should be one of the icon filenames defined in the application's `.appxmanifest` file.
 
-Summarizing the above, using `target` attribute it is possible to:
+Summarizing the above... using the `target` attribute it is possible to:
```
Comment (omefire) [inline www/docs/en/6.x/config_ref/images.md]: why '...' instead of ',' ?

**Comment by omefire** [comment]
Changes related to telemetry are definitely part of this snapshot.
Thanks Raghav!

**Comment by omefire** [comment]
However, I don't see the privacy policy changes: ```www/privacy/index.md```: https://github.com/apache/cordova-docs/commit/0d5f185c45f11a46e3dc5b720e23d4b38a40779a

**Comment by rakatyal** [comment]
@omefire: This is just copying current dev to latest version on the docs and hence you only see those changes. The privacy policy changes will take effect when I deploy the site.
Also I am not sure I can answer/change your other comments since I didn't make those changes. Asking those in the PRs on the dev branch might be a good idea.

**Comment by riknoll** [comment]
LGTM! Checked to make sure all changes were to `.md` files in 6.x and that seems to be the case.

**Comment by omefire** [comment]
LGTM! 
Changes to CLI reference are effectively in this PR. 
I'm OK with not making any changes related to my earlier comments as those were separate PRs. 

Thanks for doing this, @rakatyal

**Pull request closed without merging**